content/200-orm/500-reference/100-prisma-schema-reference.mdx

```diff
@@ -3037,35 +3040,127 @@ model User {
 </TabItem>
 </TabbedContent>
 
+##### Generate `cuid2` values as IDs
+
+<TabbedContent code>
+<TabItem value="Relational databases">
+
+```prisma
+model User {
+  id   String @id @default(cuid(2))
+  name String
+}
+```
+
+</TabItem>
+<TabItem value="MongoDB">
+
+```prisma
+model User {
+  id   String @id @default(cuid(2)) @map("_id")
+  name String
+}
+```
+
+</TabItem>
+</TabbedContent>
+
 ### `uuid()`
 
-Generate a globally unique identifier based on the [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) spec, version 4 (random).
+Generate a globally unique identifier based on the [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) spec. Prisma ORM supports versions 4 (default) and 7.
 
 #### Remarks
 
-- Compatible with `String`
+- Compatible with `String`.
 - Implemented by Prisma ORM and therefore not "visible" in the underlying database schema. You can still use `uuid()` when using [introspection](/orm/prisma-schema/introspection) by [manually changing your Prisma schema](/orm/prisma-client/setup-and-configuration/custom-model-and-field-names) and [generating Prisma Client](/orm/prisma-client/setup-and-configuration/generating-prisma-client), in that case the values will be generated by Prisma ORM's [query engine](/orm/more/under-the-hood/engines).
+- For **relational databases**: If you do not want to use Prisma ORM's `uuid()` function, you can use [the native database function with `dbgenerated`](#override-default-value-behavior-for-supported-types).
+- For **MongoDB**: `uuid()` does not generate a valid `ObjectId`. You can use [`@db.ObjectId` syntax](#generate-objectid-as-ids-mongodb-only) if you want to use `ObjectId` in the underlying database. However, you can still use `uuid()` if your `_id` field is not of type `ObjectId`.
 
-<Admonition type='info'>
+#### Examples
 
-**Note (Relational databases)**: If you do not want to use Prisma ORM's `uuid()` function, you can use [the native database function with `dbgenerated`](#override-default-value-behavior-for-supported-types).
+##### Generate `uuid()` values as IDs using UUID v4
```

Comment:
nit: you removed the parens `()` above, but kept them here and below. Could we standardize?

Reply:
nice catch! looks like we used `()` before, so I adjusted the instanced that didn't have them above 🙏